app/models/booking.rb

```diff
@@ -1,4 +1,6 @@
 class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :boat
+  validates :status, confirmation: true
```

Comment:
Please check [confirmation: true](https://guides.rubyonrails.org/active_record_validations.html#confirmation) documentation. This is not how this validator works. It's used when the input field must be confirmed:
e.g.:

password + repeat password. That's when you should you `confirmation: true`